pkgs/tools/graphics/grim/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, cairo, libjpeg, meson, ninja, wayland, pkgconfig, wayland-protocols }:
+
+stdenv.mkDerivation rec {
+  name = "grim-${version}";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "emersion";
+    repo = "grim";
+    rev = "v${version}";
+    sha256 = "1mpmxkzssgzqh9z263y8vk40dayw32kah66sb8ja7yw22rm7f4zf";
+  };
+
+  buildInputs = [
+    cairo
+    libjpeg
+    meson
+    ninja
+    pkgconfig
```

Comment:
meson, ninja and pkg-config should be nativeBuildInputs

Reply:
Done!